.eleventy.js

```diff
@@ -5,20 +5,21 @@ const fs = require("fs");
 // Import filters
 const dateFilter = require('./src/filters/date-filter.js');
 const markdownFilter = require('./src/filters/markdown-filter.js');
+const permalinkFilter = require('./src/filters/permalink-filter.js');
```

Comment:
Importing the new module, now that we've moved the function into an external file (like the other custom filters in ./src/filters/*.js).